hermes/probe/create/create.go

```diff
@@ -0,0 +1,186 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Author: Alicja Kwiecinska, GitHub: alicjakwie
+//
+// create contains all of the logic necessary to create the files with randomized contents in GCS
+package create
+
+import (
+	"context"
+	"crypto/sha1"
+	"fmt"
+	"io"
+	"math/rand"
+	"time"
+
+	"cloud.google.com/go/storage"
+	"github.com/google/cloudprober/logger"
+	"github.com/googleapis/google-cloud-go-testing/storage/stiface"
+	"github.com/googleinterns/step224-2020/hermes/probe"
+	"github.com/googleinterns/step224-2020/hermes/probe/metrics"
+
+	pb "github.com/googleinterns/step224-2020/hermes/proto"
+)
+
+const (
+	FileNamePrefixSize      = 9    // len("Hermes_ID")
```

Comment:
should the variable be declared in the create function and exported or should it be declared outside of the function body?

Reply:
If the length is required elsewhere, make it a package-level variable.
If it's only required in Create(), then move it there (closest to its usage).

Reply:
we need it in create_test.go

Reply:
Acc (caught this in my tests) var  FileNamePrefixLength = len(fmt.Sprintf(FileNameFormat, 0, ""))-1 as this prefix has two __ so it's of the form Hermes_00_ instead of Hermes_00

Reply:
acc we only need FileNamePrefixLength in create_test so only added it there and made it non-exported